packages/amplication-server/prisma/migrations/20201006094009/README.md

```diff
@@ -0,0 +1,142 @@
+You can check out the [state of the schema](./schema.prisma) after the migration.
+
+## Database Steps
+
+```sql
+CREATE TYPE "public"."EnumDeploymentStatus" AS ENUM ('Completed', 'Waiting', 'Active', 'Delayed', 'Failed', 'Paused')
+
+CREATE TABLE "public"."Environment" (
+"id" text   NOT NULL ,
+"createdAt" timestamp(3)   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+"updatedAt" timestamp(3)   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+"appId" text   NOT NULL ,
+"name" text   NOT NULL ,
+"description" text   ,
+"address" text   NOT NULL ,
+PRIMARY KEY ("id")
+)
+
+CREATE TABLE "public"."Deployment" (
+"id" text   NOT NULL ,
+"createdAt" timestamp(3)   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+"userId" text   NOT NULL ,
+"buildId" text   NOT NULL ,
+"environmentId" text   NOT NULL ,
+"status" "EnumDeploymentStatus"  NOT NULL ,
+"message" text   ,
+"actionId" text   NOT NULL ,
+PRIMARY KEY ("id")
+)
+
+CREATE UNIQUE INDEX "Environment.appId_name_unique" ON "public"."Environment"("appId", "name")
+
+ALTER TABLE "public"."Environment" ADD FOREIGN KEY ("appId")REFERENCES "public"."App"("id") ON DELETE CASCADE ON UPDATE CASCADE
+
+ALTER TABLE "public"."Deployment" ADD FOREIGN KEY ("userId")REFERENCES "public"."User"("id") ON DELETE CASCADE ON UPDATE CASCADE
+
+ALTER TABLE "public"."Deployment" ADD FOREIGN KEY ("buildId")REFERENCES "public"."Build"("id") ON DELETE CASCADE ON UPDATE CASCADE
+
+ALTER TABLE "public"."Deployment" ADD FOREIGN KEY ("environmentId")REFERENCES "public"."Environment"("id") ON DELETE CASCADE ON UPDATE CASCADE
+
+ALTER TABLE "public"."Deployment" ADD FOREIGN KEY ("actionId")REFERENCES "public"."Action"("id") ON DELETE CASCADE ON UPDATE CASCADE
+```
+
+## Changes
+
+```diff
+diff --git schema.prisma schema.prisma
+migration 20201004110208..20201006094009
+--- datamodel.dml
++++ datamodel.dml
+@@ -1,7 +1,7 @@
+ datasource db {
+   provider = "postgresql"
+-  url = "***"
++  url = "***"
+ }
+ generator client {
+   provider        = "prisma-client-js"
+@@ -52,8 +52,9 @@
+   commits       Commit[]
+   lockedEntitis Entity[]
+   builds        Build[]
++  deployments   Deployment[]
+   @@unique([accountId, organizationId])
+ }
+ model UserRole {
+@@ -81,8 +82,9 @@
+   roles          AppRole[]
+   commits        Commit[]
+   builds         Build[]
++  environments Environment[]
+   @@unique([organizationId, name])
+ }
+ model AppRole {
+@@ -302,12 +304,13 @@
+   Success
+ }
+ model Action {
+-  id        String       @id @default(cuid())
+-  createdAt DateTime     @default(now())
+-  steps     ActionStep[]
+-  builds    Build[]
++  id          String       @id @default(cuid())
++  createdAt   DateTime     @default(now())
++  steps       ActionStep[]
++  builds      Build[]
++  deployments Deployment[]
+ }
+ model ActionStep {
+   id          String           @id @default(cuid())
+@@ -361,6 +364,47 @@
+   action         Action          @relation(fields: [actionId], references: [id])
+   actionId       String
++  deployments Deployment[]
```

Comment:
Do we have to have this field?

Reply:
I have no idea why this field appears here... in the actual schema.prisma file, the ActionStep model does not have this field